packages/strapi/bin/strapi-start.js

```diff
@@ -73,14 +73,24 @@ module.exports = function() {
       setFilesToWatch(process.cwd());
 
       if (cluster.isMaster) {
-        cluster.on('message', () => {
+        cluster.on('message', (worker, message) => {
```

Comment:
Why are you not using a `switch(){...}` on the `message` variable instead of listening to the event two times?